### PR TITLE
ui: Fix bug and improve status message

### PR DESCRIPTION
**Pull request by indradhanush**
Follow up to #45619 for post merge review.

## Test plan

Tested locally.

**Message on hover**

image

**Updated status message**

image

### CODE REVIEW

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.00% (+0.10 kb) | 0.00% (+0.10 kb) | 0.00% (0.00 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/ig/ui-status-messages-fix/compare-report.html;tab=live_object) for a full comparison between the commits 5bc7802 and cb377c6 or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

File: client/web/src/nav/StatusMessagesNavItem.tsx [hunk 1]
```diff
@@ -233,7 +236,7 @@ export const StatusMessagesNavItem: React.FunctionComponent<React.PropsWithChild
                             <StatusMessagesNavItemEntry
                                 key={status.message}
                                 message={status.message}
-                                title="Git updates disabled"
+                                title="Automatic code sync disabled"
```
Comment (mrnugget) [inline client/web/src/nav/StatusMessagesNavItem.tsx:239]:
```suggestion
                                title="Code syncing disabled"
```

**Review by mrnugget — COMMENTED** [review]
Another thought: did you add the new message to the storybook for statusmessages? `StatusMessagesNavItem.story.tsx` has all status messages

**Review by mrnugget — APPROVED** [review]

**Review by sashaostrikov — APPROVED** [review]
LGTM!

**Comment by indradhanush** [comment]
Storybook:

image

**Pull request merged**